CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased](https://github.com/pusher/chatkit-android/compare/v1.8.4...HEAD)
 
+### Fixed
+
+- Remove accidentally added deprecation annotations on `subscribeRoomMultipart`
```

Comment:
```suggestion
- Remove accidentally added deprecation annotations on `subscribeRoomMultipart` that accept `RoomConsumer` as opposed to `RoomListeners`.
```
to be more specific as not all `subscribeRoomMultipart` got accidentally deprecated and for consistency with the rest of the changelog to end with a dot.